### ISSUE TITLE
Add Setting for Verbose Notification Logging

### ISSUE THREAD

**Issue by imran0** [Fallenbagel/jellyseerr#1134]
### Description

I’d like to request a feature to enable verbose logging for notifications sent via Jellyseer. The logs should include the following details for each notification:

- **Username** of the recipient  
- **Notification channel** (e.g., email, push notification, etc.)  
- **Notification content or type** (e.g., "Request Available")  

### Desired Behavior

- Provide a **toggle or configuration setting** to enable/disable verbose logging for notifications to avoid overwhelming the logs in production environments.  
- When enabled, each notification should log the following details as a single line in the

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Schrottfresser** [comment]
Would it be find to do notification logging only at DEBUG log level? A separate toggle seems overkill for me.

**Comment by gauthier-th** [comment]
> Would it be find to do notification logging only at DEBUG log level? A separate toggle seems overkill for me.

Yes totally fine. IMO a toggle is not useful for this, just log everything to debug.

**Comment by ricardoespsanto** [comment]
why not a configuration asking the user what log level do they want to see logged?
